ts/Core/Axis/ScrollbarAxis.ts

```diff
@@ -75,6 +75,13 @@ class ScrollbarAxis {
                     this: Highcharts.Scrollbar,
                     e: Highcharts.ScrollbarChangedEventObject
                 ): void {
+                    // #13473 Bars shrink if min is not set while scrolling
+                    if ((this.chart.options.chart?.type === 'column' ||
+                        this.chart.options.chart?.type === 'bar') &&
+                        !defined(axis.options.min)
```

Comment:
Ouch, I'm afraid this is a bit more tricky 😅 
Checking for `chart.type` is not enough. This option can be set to `line` but user may add `column` type series: `series: [{ type: 'column }]`. 

Checking for every series is rather cumbersome. I think (and might be wrong!) this is related to `series.threshold` and `Axis.threshold`.

Just a side note, perhaps you already know that, but you can use `series.is('column')` to check if a series is type of `column` or based on `column` (like `bar`, `columnrange` itp).